pulsar-broker/src/main/java/org/apache/pulsar/broker/web/WebService.java

```diff
@@ -159,11 +159,12 @@ public void addServlet(String path, ServletHolder servletHolder, boolean require
                 context.setAttribute(key, value);
             });
         }
+        ExceptionHandler handler = new ExceptionHandler();
```

Comment:
we can push this "new" inside the "if" block, this way we can save us from creating it if not needed

Reply:
So quick review, many thanks to you ,Enrico.

Reply:
I will update soon